doc/developers-guide/Hooks-and-handlers.md

```diff
@@ -91,9 +80,9 @@ The right way to use hooks is therefore:
 * pass the modified accumulator on
 
 Handlers should store their return values in the accumulator; there are three ways to do it:
-* if it is a one-off value which is to be discarded later use `mongoose_acc:put(result, Value, Acc)`
+* if it is a one-off value whch doesn't need to be passed on along with the accumulator (can be overwritten any time), `mongoose_acc:put(result, Value, Acc)`
```

Comment:
if it is a one-off value whch doesn't need to be passed on along with the accumulator (can be overwritten any time), use `mongoose_acc:put(result, Value, Acc)`